magpie/api/webhooks.py

```diff
@@ -53,6 +53,35 @@ def process_webhook_requests(action, params, update_user_status_on_error=False):
         pool.starmap_async(send_webhook_request, args)
 
 
+def replace_template(param_name, param_value, payload):
+    """
+    Replace each instances of a template parameter by its corresponding value.
+
+    :param param_name: name of a template parameter
+    :param param_value: value of a template parameter
+    :param payload: structure containing the data to be processed by the template replacement
+    :return: structure containing the data with the replaced template parameters
+    """
+    if isinstance(payload, dict):
+        replace_dict = payload.copy()
```

Comment:
why not simply : 
`return {replace_template(param_name, param_value, k):replace_template(param_name, param_value, v) for k, v in payload.items()}` ?

Reply:
You're right, this is way cleaner! I changed it for your suggestion.